strapi/strapi
# Feature/asset table for media library

### What does it do?
In media library page, we added a feature that allows users to take a quick look at all assets as a list.
Users can press a button to switch between card view and list view. The list view shows all actions which can be performed in one/two clicks. Additionally, it shows all asset related information which helps get a quick overview

### Why is it needed?
It saves users time by providing a quick overview of each asset.

### How to test it?
- Click on Show Asset List to see the list
- Freely interact which each asset in the list 
- Switch back to card view by clicking on the Show Asset Card
<img width="660" alt="Screen Shot 2022-06-02 at 1 15 23 PM" src="https://user-images.githubusercontent.com/88243340/171730209-ad985763-8295-4e70-9494-4fbbe0ee87d0.png">


### Design document:
https://docs.google.com/document/d/1CQkRTItfi7DMvEbKkMrG7JYU-3Vji5zcuJIMWqXhRvw/edit?usp=sharing

### Related issue(s)/PR(s)

This is a new feature.
